src/app/containers/BulletedList/index.jsx

```diff
@@ -16,7 +19,24 @@ const StyledGridItemMedium = styled(GridItemMedium)`
 `;
 
 const BulletedListContainer = ({ blocks, className, ...rest }) => {
+  const linkBlock = blocks.find(element => {
+    const locator = pathOr(
+      null,
+      ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'locator'],
+      element,
+    );
+    return locator;
+  });
```

Comment:
Ramda's `pathOr` uses currying, so I think you could reduce this to:

```javascript
const linkBlock = blocks.find(pathOr(
  null,
 ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'locator']
));
```

You could also use `path` instead:

```javascript
const linkBlock = blocks.find(path(
 ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'locator']
));
```

This will return `undefined` rather than `null`, but from how it's being used, that shouldn't matter

Reply:
Thanks, the `blockId `still uses `pathOr` which returns null for blocks that don't have ID's - reckon thats fine for us to keep as is?

Reply:
The `pathOr` returns the string `'null'` rather than `null` - guessing that's intended?   If you want the string `'null'` then yeah, `pathOr` is ideal 🙂